### DISCUSSION TITLE
Doc Update Recommendation request to help maintain this Gem

### DISCUSSION THREAD

**Discussion by sir-george2500** [jaredh159/tailwind-react-native-classnames#296] [Announcements]
Hi there! I recently discovered your project and I'm eager to contribute. However, I'm feeling a bit lost on where to start with contributing and maintaining this library, especially since I'm new to open source. Are there any documentation or guidelines available to assist beginners like me? I'm particularly interested in helping out because I'm using this library in my own app and would like to contribute to its development and maintenance.

**Comment by jaredh159** [comment]
hi there, thanks for reaching out. appreciate your desire and willingness to help.

i don't have much in the way of documentation for contributing, you're right. i'm not sure exactly if or when i will add some. the library is fairly small, and usually the best way to get involved would be to fix a bug or add a feature in a PR (opening a discussion might be good before implementing a new feature though, because i try to keep the library pretty small and focused).  

that said, we don't have a ton of open issues right now, but if you were interested in getting involved, you could try finishing up #293 which is moderately straightforward.
